### PR TITLE
Apps versioning

### PR DESCRIPTION
**Pull request by manucamejo**
## Motivation

We needed to know which version we have in each server in order to prevent a bad experience in the users.

## Summary of changes

- Add servers table
- Add arena version endpoint

## Checklist
- [X] Tested the changes locally.
- [ ] Reviewed the changes on GitHub, line by line.
- [ ] This change requires new documentation.
  - [ ] Documentation has been added/updated.

### CODE REVIEW

File: apps/arena/mix.exs [hunk 1]
```diff
@@ -4,7 +4,7 @@ defmodule Arena.MixProject do
   def project do
     [
       app: :arena,
-      version: "0.1.0",
+      version: "0.1.14",
```
Comment (AminArria) [inline apps/arena/mix.exs:7]: Arbitrary number? Jajaja

File: apps/configurator/lib/configurator_web/controllers/arena_server_controller.ex [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    end
+  end
+
+  def delete(conn, %{"id" => id}) do
+    arena_server = Configuration.get_arena_server!(id)
+    {:ok, _arena_server} = Configuration.delete_arena_server(arena_server)
+
+    conn
+    |> put_flash(:info, "Arena server deleted successfully.")
+    |> redirect(to: ~p"/arena_servers")
```
Comment (Nico-Sanchez) [inline apps/configurator/lib/configurator_web/controllers/arena_server_controller.ex:60]: We should put the delete_arena_server/1 function in a case statement to handle the :error clause.

File: apps/gateway/lib/gateway/controllers/arena_servers_controller.ex [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+      |> Enum.map(fn arena_server ->
+        %{
+          id: arena_server.id,
+          ip: arena_server.ip,
+          url: arena_server.url,
+          name: arena_server.name,
+          status: arena_server.status,
+          environment: arena_server.environment
+        }
+      end)
```
Comment (Nico-Sanchez) [inline apps/gateway/lib/gateway/controllers/arena_servers_controller.ex:22]: This isn't necessary if you add `@derive Jason.Encoder` right before defining the ArenaServer schema.
I'll add a comment where you should put it for more clarity.

File: apps/game_backend/lib/game_backend/arena_servers/arena_server.ex [hunk 4]
```diff
@@ -0,0 +1,24 @@
+defmodule GameBackend.ArenaServers.ArenaServer do
+  @moduledoc """
+  ArenaServers
+  """
+  use GameBackend.Schema
+  import Ecto.Changeset
+
+  schema "arena_servers" do
```
Comment (Nico-Sanchez) [inline apps/game_backend/lib/game_backend/arena_servers/arena_server.ex:8]: You can add it here (see comment below):
```suggestion
  @derive Jason.Encoder
  schema "arena_servers" do
```

File: apps/arena/mix.exs (hunk 1 above)
Comment (manucamejo) [reply apps/arena/mix.exs:7]: Yeah, but we need to sync front and back when this pr gets merged

**Review by AminArria — DISMISSED** [review]
LGTM

**Review by Nico-Sanchez — CHANGES_REQUESTED** [review]

File: apps/gateway/lib/gateway/controllers/arena_servers_controller.ex (hunk 3 above)
Comment (manucamejo) [reply apps/gateway/lib/gateway/controllers/arena_servers_controller.ex:22]: nice catch

**Review by Nico-Sanchez — DISMISSED** [review]

**Review by agustinesco — APPROVED** [review]
LGTM!

**Review by agustinesco — APPROVED** [review]
LGTM

**Review by AminArria — APPROVED** [review]

**Pull request merged**